alembic/versions/8bca5e25546e_rename_tables.py

```diff
@@ -0,0 +1,90 @@
+"""Rename tables
+
+Revision ID: 8bca5e25546e
+Revises: 891900abdb6b
+Create Date: 2020-08-10 10:48:48.817288
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '8bca5e25546e'
+down_revision = 'aa34d6c338e6'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # rename processed_model_run_files -> processed_model_run_urls.
```

Comment:
Are these comments autogenerated? if so that's awesome

Reply:
I wish. The entire migration was painfully hand-crafted. Alembic doesn't autogenerate renaming of stuff. :(